src/aya/lifecycle.md

```diff
@@ -0,0 +1,68 @@
+# Program Lifecycle
+
+## BPF Object Lifecycle
```

Comment:
I'd skip this section, fds are terrible I don't think we need to teach users to reason in terms of them